packages/cfd/src/Containers/mt5-create-password/mt5-create-password.tsx

```diff
@@ -68,15 +68,20 @@ const MT5CreatePassword = ({
                 validateForm,
             }) => (
                 <form onSubmit={handleSubmit}>
-                    <div className='mt5_password'>
-                        <div
-                            className='cfd-password-modal__content dc-modal__container_cfd-password-modal__body'
-                            data-testid='dt_create_password'
-                        >
+                    <div
+                        className='cfd-password-modal__content dc-modal__container_cfd-password-modal__body'
+                        data-testid='dt_create_password'
+                    >
+                        <div className='cfd-password-modal__create-password-body mt5-migration-modal__mobile-container'>
```

Comment:
please check the classnames. why are we using mt5-migration-modal__mobile-container classnames?

Reply:
removed was added as it was used in previous PR, now not needed